wcfsetup/install/files/lib/system/cache/builder/AbstractLegacyCacheBuilder.class.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace wcf\system\cache\builder;
+
+use wcf\system\cache\CacheHandler;
+use wcf\system\exception\SystemException;
+use wcf\system\SingletonFactory;
+
+/**
+ * Legacy implementation of the ICacheBuilder interface that has been migrated to a new EagerCache or AsyncCache.
+ *
+ * @author      Olaf Braun
+ * @copyright   2001-2025 WoltLab GmbH
+ * @license     GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ *
+ * @since       6.2
+ * @deprecated  6.2
```

Comment:
No need for the indentation, it’s messed up anyway and nobody cares. Save yourself some time and just use a single space to separate values.